catanatron/models/actions.py

```diff
@@ -28,6 +29,21 @@ class ActionType(Enum):
 
 Action = namedtuple("Action", ["player", "action_type", "value"])
 
+def year_of_plenty_possible_actions(player, resource_deck):
+    possible_combinations = set()
+    actions = []
+    for first_card in Resource:
+        for second_card in Resource:
+            if resource_deck.can_draw(1, first_card) and resource_deck.can_draw(1, second_card) and (second_card, first_card) not in possible_combinations:
+                possible_combinations.add((first_card, second_card))
```

Comment:
this is just to avoid adding a duplicate action... For example, if we already generated an action that can pick ORE and WHEAT, we don't want another action with potential to pick WHEAT and ORE.

Reply:
Perfect! Makes sense! :fire: A `set` seems like the right data structure.